tests/drivers/adc/adc_api/src/test_adc.c

```diff
@@ -219,6 +219,15 @@
 #define ADC_1ST_CHANNEL_ID	4
 #define ADC_2ND_CHANNEL_ID	5
 
+#elif defined(CONFIG_BOARD_NPCX7M6FB_EVB)
```

Comment:
Need to add `adc` to the board's supported list yaml to make sanitycheck build this test

Reply:
Got it. Thanks for reminding that.

Reply:
Done.